.gitignore

```diff
@@ -4,3 +4,4 @@ node_modules/
 .idea/**
 .husky
 .nyc_output
+.DS_Store
```

Comment:
Better configure it's globally once in your system:
https://dev.to/lpheller/quickly-ignore-dsstore-files-globally-ecd